templates/docker-compose.template

```diff
@@ -65,12 +62,6 @@ services:
       - "80:5000"
     cap_add:
       - SYS_RAWIO
-    devices:
-      - {{I2C_DEVICE}}:{{I2C_DEVICE}}
```

Comment:
do we not need to map these in to the container?

Reply:
From now on the gatewayrs (helium-miner) container is privileged. So It doesn't need explicit device definitions upfront. Same thing is valid for any privileged container, e.g. diagnostics.

Reply:
Ah ok fair enough